src/raftstore/store/worker/pd.rs

```diff
@@ -696,10 +694,9 @@ impl<T: PdClient> Runnable<Task> for Runner<T> {
                     peer_stat.last_read_bytes = peer_stat.read_bytes;
                     peer_stat.last_read_keys = peer_stat.read_keys;
                     peer_stat.last_report_ts = time_now_sec();
-                    last_report_ts = cmp::max(
-                        last_report_ts,
-                        peer_stat.last_report_ts - self.region_heartbeat_interval,
-                    );
+                    if last_report_ts == 0 {
+                        last_report_ts = self.start_ts;
+                    }
```

Comment:
maybe we need update the `start_ts` if `last_report_ts != 0`?  because the leader may be changed to another for a while and then change back.

Reply:
I think it is unnecessary.
For example, assume peer A does heartbeat at t1, then transfer leader to peer B at t2, and leader immediately transferred back to peer A at t3.
In this case, when peer A does heartbeat at t3, its `last_report_ts` will be set to `peer_stat.last_report_ts` (t1), and its reported I/O information will also match this time window (t1->t3). (Peer A still records the write information even it was not a leader.)

Reply:
got it